docs/reference/src/main/asciidoc/injection.asciidoc

```diff
@@ -551,6 +558,12 @@ Annotation qualifier = synchronously ?
 PaymentProcessor p = anyPaymentProcessor.select(qualifier).get().process(payment);
 ----------------------------------------------------------------------------------
 
+[NOTE]
```

Comment:
Hmm, I disagree. 
In `4.4` we don't say a word about implementing a qualifier in order to use it programmatically.
Here, on the contrary, we do, so I think we should straight away say that you needn't implement all of them.